### ISSUE TITLE
Having Transcended and Confix enabled messes up dialogue

### ISSUE THREAD

**Issue by Deer-Trap** [Defaultplayer001/Deus-Ex-Universe-Community-Update-#33]
If both of these options are enabled when installing CU then the dialogue is totally ruined and random audio lines play, this is apparent as early as the docks in the first level.

If CU is installed with only Transcended checked everything works. 

(This issue exists with at least the last two versions released)

EDIT: I fixed this by manually downloading confix and overwriting the files in the system folder. I'm not sure why this was necessary. 

Not sure if the CU provided Confix files are corrupted or something?

**Comment by VermilionCosmos** [comment]
Was coming here to also report this. Noticeable on first level. Fixing it just takes reinstalling Confix manually.

**Comment by tsukihidaze** [comment]
Commenting here to say that this is also an issue on the latest GOG version of Deus Ex GOTY. Installing Transcended by itself seems to work fine in the mean time (I have not tried what VermilionCosmos said).

**Comment by Defaultplayer001** [comment]
I can't seem to recreate this on the latest version (3.3)? Make sure your on a clean install as well, any confliction conversation files could cause this mismatch. I switched up how I did things in recent versions to prevent this issue even being possible hopefully! (DX is very finicky about remote loading con files if there's a local copy present, simply deleting that makes things much more cooperative. I just move the vanilla con files to a remote location as well.)
